platform/darwin/docs/guides/Predicates and Expressions.md

```diff
@@ -10,7 +10,9 @@ This document discusses the specific subset of the predicate and expression
 syntax supported by this SDK. For a more general introduction to predicates and
 expressions, consult the
 _[Predicate Programming Guide](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/Predicates/AdditionalChapters/Introduction.html)_
-in Apple developer documentation.
+in Apple developer documentation. For additional detail on how this SDK has
+extended the `NSExpression` class, consult the [`NSExpression(MGLAdditions)`](./Categories/NSexpression(MGLAdditions).html)
```

Comment:
Is that URL with the lower case "e" (`./Categories/NSexpression(MGLAdditions).html`) correct?

Reply:
Good catch. This is my fault for fiddling around with the linking too much. The link still works, but I'll change it for correctnesses sake.